src/Microsoft.DotNet.GitHub.IssueLabeler/appsettings.Development.json

```diff
@@ -0,0 +1,4 @@
+{
+  "RepoOwner": "dotnet",
```

Comment:
why do we need an extra file ?

Reply:
This config file is used in local development only. In production the configuration settings come from per-instance environment variable settings.